src/app/app.module.ts

```diff
@@ -60,6 +61,17 @@ export function restangularProviderConfigurer(
   });
 }
 
+export const RESTANGULAR_MAPPER = new InjectionToken<any>('restangularMapper');
```

Comment:
Change this to:

```
export const RESTANGULAR_MAPPER = new InjectionToken<Restangular>('MapperRestangular');
```

Reply:
Actually the token value can be `restangularMapper` that's fine, just change the type from `any` to `Restangular` as this token is apparently supposed to be typed.